pynsee/utils/init_conn.py

```diff
@@ -61,40 +63,33 @@ def init_conn(insee_key, insee_secret, http_proxy="", https_proxy=""):
         },
         index=[0],
     )
+
     d.to_csv(pynsee_credentials_file)
 
-    keys = _get_credentials()
+    _get_credentials()
 
-    insee_key = keys["insee_key"]
-    insee_secret = keys["insee_secret"]
+    insee_key = pynsee._config["insee_key"]
+    insee_secret = pynsee._config["insee_secret"]
 
-    token = None
-    try:
-        token = _get_token_from_insee(insee_key, insee_secret)
-    except:
-        pass
+    token = _get_token(insee_key, insee_secret)
 
-    if token is None:
+    if not token:
         raise ValueError(
-            "!!! Token is missing, please check insee_key and insee_secret are correct !!!"
-        )
-    else:
-        logger.info("Token has been created")

```

Comment:
Should we keep a trace in log (I've a project which was repeating those calls and this was helpful). Would be better inside `_get_token_from_insee` instead, though.